pkg/k8s/constants.go

```diff
@@ -113,6 +114,9 @@ var (
 		OrgIDAnnotation,
 	}
 
+	// Internal Annotation to force reconciliation
+	InternalForceReconcileAnnotation = CNRMInternalGroup + "/reconcile-cookie"
```

Comment:
this may be overkill but do we want to prevent modifying this anon for non test usage?

Reply:
I like your change to make it `test`, it clearly signals that this should not be relied on in prod.  If we discover that people are (ab)using it, that's interesting info and we should probably implement a similar production-suitable feature (which might not actually look very different, but .... shrug)